addons/binding/org.openhab.binding.hdpowerview/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="hdpowerview"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <bridge-type id="hub">
+        <label>PowerView Hub</label>
+        <description>Hunter Douglas PowerView Hub</description>
+        
+        <config-description>
+            <parameter name="ipAddress" type="text" required="true">
+                <label>IP</label>
+                <description>The IP of the PowerView Hub.</description>
+            </parameter>
+            <parameter name="refresh" type="integer" required="false">
+                <label>Refresh Interval</label>
+                <description>The number of milliseconds between fetches of the PowerView Hub shade state.</description>
+                <default>60000</default>
+            </parameter>
+        </config-description>
+    </bridge-type>
+    
+    <thing-type id="shade">
+        <label>PowerView Shade</label>
+        <description>Hunter Douglas PowerView Shade</description>
+        <channels>
+            <channel id="position" typeId="shade-position"/>
+            <channel id="vane" typeId="shade-vane"/>
+            <channel id="batteryLow" typeId="shade-batteryLow"/>
+        </channels>
+        <config-description>
+            <parameter name="id" type="text" required="true">
+                <label>ID</label>
+                <description>The ID of the PowerView Shade.</description>
+            </parameter>
+        </config-description>
+    </thing-type>
+
+    <channel-type id="shade-position">
+        <item-type>Rollershutter</item-type>
+        <label>Position</label>
+        <description>The vertical position of the shade</description>
+    </channel-type>
+    <channel-type id="shade-vane">
+        <item-type>Dimmer</item-type>
+        <label>Vane</label>
+        <description>The opening of the slats in the shade</description>
+    </channel-type>
+    <channel-type id="shade-batteryLow">
+        <item-type>Switch</item-type>
+        <label>Battery Low</label>
+        <description>Indicates the battery state of the shade</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <thing-type id="scene">
```

Comment:
Scenes do not really feel like a good fit for a "Thing" of its own. They are rather a functionality of the hub itself and hence I would expect them to be a channel on the "hub thing".
As they do not even have a state on their own, a single "scene" channel (of type number or string) would actually suffice I would think (although it leaves the question of how best to provide the stateDescription, in case the binding could provide this).